### ISSUE TITLE
The home page of this repo seems have a broken image.

### ISSUE THREAD

**Issue by doggy8088** [Overmiind/Puppeteer-sharp-extra#68]
https://github.com/Overmiind/Puppeteer-sharp-extra?tab=readme-ov-file

![image](https://github.com/user-attachments/assets/8cffd89f-8aff-4124-a0a4-5d5f22965b12)

**Comment by KoalaBear84** [comment]
Yes, and I'm scared it might stay this way because of the unfair war in Ukraine. ❤

**Comment by Overmiind** [comment]
Fixed, thanks
